package.json

```diff
@@ -41,6 +40,9 @@
     "reduce-reducers": "^1.0.4",
     "typescript": "4.0.3"
   },
+  "devDependencies": {
+    "firebase-tools": "^10.0.1"
+  },
```

Comment:
Don't people install this globally? Maybe it shouldn't be a dep at all?

Reply:
I never install npm packages globally. With this, people can just `yarn firebase` instead.